articles/basic-code-elements-introduction.html

```diff
@@ -174,36 +174,39 @@ <h3>A Simple Go Demo Program</h3>
 Line <i>5</i> declares a named constant, <code>MaxRand</code>.
 Line <i>14</i> and line <i>27</i> declare three variables,
 with the standard variable declaration form.
-The variable <code>i</code>, <code>x</code> and <code>y</code> at line <i>29</i>
-are declared with the short variable declaration form.
-Compilers will deduce that the types of <code>i</code>, <code>num</code>,
-<code>x</code> and <code>y</code> are all <code>int</code>.
+Variables <code>i</code> at line <i>15</i>, <code>x</code> and <code>y</code> at line <i>29</i>
+are declared with the short variable declaration form. We have specified the type
+for variables <code>a</code> and <code>b</code> as <code>int</code>.
+Go compiler will deduce that the types of <code>i</code>, <code>num</code>,
+<code>x</code> and <code>y</code> are all <code>int</code> because we initialized them
+with <code>int</code> literals.

```

Comment:
There are no the "int literal" terminology.
So I prefer to change the last line to
"because we initialize them with integer literals"
or
"because they are initialized with integer literals".